app/assets/javascripts/application.js

```diff
@@ -85,6 +85,8 @@ $(function() {
   $('a[href*=#]:not([href=#])').click(function() {
     if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
       var target = $(this.hash);
+      preventDefault()
```

Comment:
well, I guess removing the `e` didn't fix anything. Though my anchor stuff still won't work without this line... Halp